src/gui/windows/debug.zig

```diff
@@ -48,11 +50,17 @@ fn flawedRender() !void {
 		y += 8;
 		// TODO: packet loss
 		// TODO: Protocol statistics(maybe?)
+                try draw.print("Connections number: {}", .{main.game.world.?.manager.connections.items.len},  0, y, 8, .left);
+                y += 8;
+                try draw.print("PacketsSent number: {}", .{Connection.packetsSent.value},  0, y, 8, .left);
+                y += 8;
+                try draw.print("PacketsResent number: {}", .{Connection.packetsResent.value},  0, y, 8, .left);
```

Comment:
That's not really a nice way to display these numbers. See #153 for how I did it in the java version.